README.md

```diff
@@ -0,0 +1,10 @@
+### `endaq-python`: A comprehensive, user-centric Python API for working with enDAQ data and devices
+
+`endaq-python` is essentially a meta-package, combining several function-specific sub-packages, and providing a central repository from which to build the documentation.
+
+This package consists of several submodules:
+* `endaq.calc` ([repo](https://github.com/MideTechnology/endaq-python-calc)): A computational backend for vibration analysis.
+* `endaq.cloud` ([repo](https://github.com/MideTechnology/endaq-python-cloud)): Tools for interacting with enDAQ Cloud services.
+* `endaq.ide` ([repo](https://github.com/MideTechnology/endaq-python-ide)): high-level utility functions to aid in importing and inspecting enDAQ IDE recording files.
+<!-- * `endaq.io` ([repo](https://github.com/MideTechnology/endaq-python-io)) -->
+* `endaq.plot` ([repo](https://github.com/MideTechnology/endaq-python-plot)): Utility functions for plotting data.
```

Comment:
I believe `endaq.plots` isn't going in the first release, so I think it should *also* be commented out like `endaq.io`

Reply:
fixed in adbbc6f50e2dd0a40d50760b7e33d392157b85d4